### ISSUE TITLE
clinic flame fails when app is using Pino

### ISSUE THREAD

**Issue by alexalmadav** [clinicjs/node-clinic#337]
## Expected Behavior
 Running clinic flame on a node.js app using [Pino logging library](https://github.com/pinojs/pino) should work. `clinic doctor` and `clinic bubbleprof` work fine

## Current Behavior
I get an error

```
Error: SonicBoom supports only file descriptors and files
    at new SonicBoom (node_modules/pino/node_modules/sonic-boom/index.js:109:11)
    at buildSafeSonicBoom (node_modules/pino/lib/tools.js:296:18)
    at Proxy.module.exports.destination (node_modules/pino/pino.js:220:12)
    at Object.<anonymous> (packages/libraries/logger/lib/sinks.js:10:23)
    at Generator.next (<anonymous>)
    at Object.<anonymous> (packages/libraries/logger/lib/default-logger.js:1)
    at Generator.next (<anonymous>)
[    ] Analysing dataTarget subprocess error, code: 1
```

The way Pino is being initialized in our libary is this
```
const pinoDest = pino.destination({
  minLength,
  sync: !ENABLE_ASYNC_LOGGING,
});


const PinoLevelToSeverityLookup = {
  debug: 'DEBUG',
  error: 'ERROR',
  fatal: 'CRITICAL',
  info: 'INFO',
  trace: 'DEBUG',
  warn: 'WARNING',
};

const defaultPinoConf = {
  formatters: {
    level(label) {
      return {
        severity: PinoLevelToSeverityLookup[label] || 'INFO',
      };
    },
    log(message) {
      return message;
    },
  },
  messageKey: 'message',
  prettyPrint: !IS_PROD,
};

const logger = pino(defaultPinoConf, pinoDest);
```

## Environment

* Clinic.js version: v11.0.0 & Doctor v9.2.1

* Node.js version: v16.13.2 & npm 8.1.2

* Browser name and version: n/a

* Operating system and version: macOS Monterey 12.2.1 on Apple M1 and Apple Intel

**Comment by RafaelGSS** [comment]
It actually is a bug inside `0x`.

```js
const http = require('http');
const pino = require('pino')

// This throws the error
const pinoDest = pino.destination({ sync: false })

const defaultPinoConf = {
  prettyPrint: true,
};

const logger = pino(defaultPinoConf);

const requestListener = function (req, res) {
  logger.info({ message: 'hello world log' })
  res.writeHead(200);
  res.end('Hello, World!');
}

const server = http.createServer(requestListener);
server.listen(8080);
```

**Comment by RafaelGSS** [comment]
I'm investigating it, meanwhile, you can use: `LOG_TO_CONSOLE=false` env variable to make it work.

**Comment by mcollina** [comment]
Fixed in 0x v5.2.1
